spec/htmlbook_spec.rb

```diff
@@ -217,6 +217,24 @@
 		html.xpath("//pre[@data-type='programlisting'][1]/@data-code-language").text.should == "php"
 	end
 
+	# Tests span nested in source block
+	it "should convert spans nested in source blocks" do
+		html = 	html = Nokogiri::HTML(convert('
```

Comment:
```suggestion
		html = Nokogiri::HTML(convert('
```
(Unless I'm missing why you'd need the double assignment?)